tests/test_handlers.py

```diff
@@ -43,6 +43,34 @@ def test_build_message_without_attachments(mocker):
     assert test_message.get_payload()[1].get_payload() == '<p>testing version: 0</p>'
 
 
+def test_build_message_with_None_attachment(mocker):
+
+    distribution_Mock = mocker.Mock()
+    distribution_Mock.version = 0
+    distributions = [distribution_Mock]
+    mocker.patch('pkg_resources.require', return_value=distributions)
+
+    message_details = MessageDetails()
+    message_details.message = 'test_message'
+    message_details.subject = 'test_subject'
+    message_details.project_name = 'testing'
+    message_details.attachments = [None]
```

Comment:
**thought**: do you need another test for empty strings?

Reply:
Actually yeah, `Path('')` becomes the current directory, which is probably a bad thing for this.